### DISCUSSION TITLE
CTkRangeSlider

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#1415] [Showcase]
# [CTkRangeSlider](https://github.com/Akascape/CTkRangeSlider)
Now you can add a **ctk based range/dual-headed slider widget** in your customtkinter app:

![Screenshot](https://user-images.githubusercontent.com/89206401/229349732-0e078b00-f2b7-46e7-9774-313090253769.jpg)

**Comment by FaridAlmaulaui1** [comment]
Hey, I really like your work and appreciate it, is there any slider widget around with ticks or a label inside the blue slider button. and if there isn't can you give me a hint on how to make it 

Thank you
